README.md

```diff
@@ -5,102 +5,165 @@
 It runs with [Prow CI](https://github.com/kubernetes/test-infra/blob/master/prow/) on a [Google Kubernetes Engine Cluster](https://cloud.google.com/kubernetes-engine/).
 It is designed to support adding more k8s providers.
 
+## How to run tests manually
+
+### Prerequisites
```

Comment:
now with this format look like we don't even need to have a paragraph for Prerequisites